lib/Activity.py

```diff
@@ -526,14 +528,18 @@ def run_stages(self, config):
         self.site_conf = SiteConfig(config)
         self.site_conf.organize_merge()
 
-        bp_config_management = config.args.get("bootprep_config_management", None)
+        bp_config_management = config.args.get("relative_bootprep_config_management", None)
         if bp_config_management:
             payload["input_parameters"]["bootprep_config_management"] = bp_config_management
 
-        bp_config_managed = config.args.get("bootprep_config_managed", None)
+        bp_config_managed = config.args.get("relative_bootprep_config_managed", None)
         if bp_config_managed:
             payload["input_parameters"]["bootprep_config_managed"] = bp_config_managed
 
+        bp_config_dir = config.args.get("relative_bootprep_config_dir", None)
```

Comment:
Not sure if this should be passed to the backend.  The backend isn't using it.  This might just be a relic of the cne-installer.